src/utils/AuthApi.test.ts

```diff
@@ -0,0 +1,49 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useAuthenticationApi } from './AuthApi';
+import { ApiResponse } from './UseAxios';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+
+const email = 'testEmail@email.com';
+const password = 'TestPassword';
+
+const httpHeaders = new Headers({
+  'Content-Type': 'json',
+  'Project-Name': 'sdc',
+});
+
+const getRequestData = {
```

Comment:
This method name is a bit weird, it's named as a function when it's really just an object.

Reply:
Renamed to `requestData`